README.md

```diff
@@ -90,20 +96,23 @@ As copyright owner, ZeroC can license Ice under different license terms, and off
 - Commercial or closed-source licenses
 
 If you license Ice under GPL v2, there is no license fee or signed license agreement: you just need to comply with the
```

Comment:
I know we still link to the GPLv2 license via the badge, but I think it'd be nice to still have a link here in this section.